docs/pages/extensibility/api.md

```diff
@@ -25,6 +25,29 @@ Each of the API endpoints are exposed through [server.go](https://github.com/lay
 
 Alternatively, [Remote Providers](./providers) can extend Meshery's endpoints behind the `/api/extensions/` endpoint.
 
+### How to get your token
+
+There are two ways to get your token:
+1. Meshery UI
+2. Meshery CLI
+
+- What are tokens?
+Tokens allow a user to authenticate with cloud apps and bypass two step verification and SSO, retrieve data from the instance through REST APIS
+An authentication token (security token) is a “trusted device” used to access an electronically restricted resource (usually an application or a corporate network). It can be seen as an electronic key that enables a user to authenticate and prove his identity by storing some sort of personal information.
+
+
+- Firstly, lets discuss how to get the token using Meshery-UI
+1. Login/Signup to the dashboard of Meshery-UI
+2. Click on Get token from the dropdown of profile section, 
+3. You can download your token from there
+
+- Secondly, you can download your token using Meshery-CLI
```

Comment:
Will it be better to phrase it as `mesheryctl` or Meshery-CLI?

Reply:
If it was "Meshery-CLI", which to @bariqhibat's point it probably isn't, you'll want to drop the hyphenation and use the indefinite article in reference to it - there is only one Meshery CLI.